client-spark/common/src/main/java/org/apache/spark/shuffle/ShuffleHandleInfo.java

```diff
@@ -25,13 +25,17 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
 import java.util.stream.Collectors;
 
 import com.google.common.annotations.VisibleForTesting;
+import org.apache.commons.collections.CollectionUtils;
+import org.eclipse.jetty.util.ConcurrentHashSet;
```

Comment:
It seems that there is a wrong quote here.

Reply:
Removed.